### PR TITLE
Feature/api integration

### PR DESCRIPTION
**Pull request by govindappaarun**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,3 +1,2 @@
-
 REACT_APP_JWT_SECRET="iloveyou"
 PORT=4000
```
Comment (nayyyhaa) [inline .env:2]: add .env file in gitignore. try not to upload this file on github.. read: https://stackoverflow.com/questions/30696930/how-to-hide-env-file-from-github

File: src/pages/cart/Cart.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  return (
+    <Wrapper>
+      <Header />
+      <main>
+        <Typography align="center" variant="h1">
+          My Cart ({state.quantity})
+        </Typography>
+        {state.cart.map((item) => (
+          <CartCard key={item._id}>
+            <div className="card-meida">
```
Comment (nayyyhaa) [inline src/pages/cart/Cart.js:42]: suggestion: you can destructure `item` beforehand for better use.

**Review by nayyyhaa — APPROVED** [review]
Only few suggestions, try to remove .env file from here. rest of the code looks good to merge.

**Review by vrinda-mahajan — APPROVED** [review]
Nothing much to suggest. The code is easily understandable. Good to merge.

**Pull request merged**